docs/OpenBSD.md

```diff
@@ -0,0 +1,78 @@
+# Getting started with Swift on OpenBSD
+
+Swift builds and runs on OpenBSD (tested on 6.8-beta), with some special considerations.
+
+## Preparing
+
+The following packages are required to build Swift. You can install these via `pkg_add`:
+
+```
+$ doas pkg_add bash cmake e2fsprogs git icu4c ninja py-six python3
+```
+
+Because LLVM is built as part of building Swift and does not include some of the patches to handle the OpenBSD library naming convention, you will need to create some symlinks:
+
+```
+$ doas ln -s /usr/lib/libc++abi.so.2.1 /usr/lib/libc++abi.so
+$ doas ln -s /usr/lib/libc++.so.4.0 /usr/lib/libc++.so
+$ doas ln -s /usr/lib/libc.so.96.0 /usr/lib/libc.so
+$ doas ln -s /usr/lib/libm.so.10.1 /usr/lib/libm.so
+$ doas ln -s /usr/local/lib/libicuuc.so.18.0 /usr/local/lib/libicuuc.so
+$ doas ln -s /usr/lib/libpthread.so.26.1 /usr/lib/libpthread.so
+```
+
+*Note: you may need to update the version numbers if necessary.*
+
+Also link `~/bin/python` to the `python2.7` binary:
+
+```
+$ doas ln -s /usr/local/bin/python2.7 ~/bin/python
+```
+
+Since the build requires significant amounts of memory at certain points, you may need to ensure both that the user you are using to build Swift has the appropriate limits set. Using the `staff` group in `login.conf` and ensuring the shell limits are raised is recommended.
```

Comment:
```suggestion
Since the build requires significant amounts of memory at certain points, you may need to ensure that the user you are using to build Swift has the appropriate limits set. Using the `staff` group in `login.conf` and ensuring the shell limits are raised is recommended.
```